faucet/src/utils.rs

```diff
@@ -0,0 +1,100 @@
+use std::{
+    fs::File,
+    io::{self, Read},
+    path::{Path, PathBuf},
+};
+
+use miden_client::{
+    client::{rpc::TonicRpcClient, Client},
+    config::{RpcConfig, StoreConfig},
+    store::{sqlite_store::SqliteStore, AuthInfo},
+};
+use miden_lib::{accounts::faucets::create_basic_fungible_faucet, AuthScheme};
+use miden_objects::{
+    accounts::{Account, AccountData},
+    assets::TokenSymbol,
+    crypto::dsa::rpo_falcon512::KeyPair,
+    utils::serde::Deserializable,
+    Felt,
+};
+
+/// Instantiates the Miden client
+pub fn build_client(database_filepath: String) -> Client<TonicRpcClient, SqliteStore> {
```

Comment:
Seeing as `Client<TonicRpcClient, SqliteStore>` is used in various places, it could be benefitial to define a newtype that makes it more succinct.